docs/specification/1.0.md

```diff
@@ -193,7 +193,7 @@ Similarly, each EHR will decide what FHIR resources to authorize and to prefetch
 
 ### Prefetch Template
 
-A _prefetch template_ is a `read` or `search` request that describes relevant data needed by the CDS Service. For example, the following is a prefetch template for hemoglobin A1c observations:
+A _prefetch template_ is a [`read`](http://hl7.org/fhir/http.html#read) or [`search`](http://hl7.org/fhir/http.html#search) request that describes relevant data needed by the CDS Service. For example, the following is a prefetch template for hemoglobin A1c observations:
```

Comment:
Hey Bryn, what do you think about adding the word FHIR to this sentence (in addition to the links), e.g.:
>A prefetch template is a FHIR read or search request. 

I think that better meets the spirit of the comment.

Reply:
Agreed and done